package.json

```diff
@@ -6,28 +6,26 @@
     "type": "git",
     "url": "https://github.com/kenchris/urlpattern-polyfill"
   },
-  "main": "./dist/index.umd.js",
-  "module": "./dist/index.js",
-  "unpkg": "./dist/index.umd.js",
-  "types": "./dist/index.d.ts",
-  "exports": "./dist/index.js",
-  "type": "module",
+  "exports": {
+    "import": "./dist/index.mjs",
+    "require": "./dist/index.js",
+    "types": "./dist/index.d.ts"
+  },
   "files": [
-    "src",
     "dist"
   ],
   "devDependencies": {
-    "ava": "^3.14.0",
-    "esm": "^3.2.25",
-    "microbundle": "^0.13.3",
-    "rimraf": "^3.0.2",
-    "rollup": "^2.35.1"
+    "ava": "3",
+    "esm": "latest",
```

Comment:
These aren't god constraints to use in a checked-in package.json. I'd use lower and upper bounds like they were before.